woocommerce/woocommerce
# [CYS] Fix the failed to load resource error in the CYS whenever the current active theme is not TT4

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/45479



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure the `WooCommerce Beta Tester` plugin is installed and activated (available on this monorepo).
2. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
3. Click on `Tools` and run the `Reset Customize Your Store` and the `Delete all products` commands.
5. Enable the TT1 theme.
6. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store`.
7. Open the inspector.
8. Click on `Start designing`.
9. Make sure you see the following modal:
<img width="860" alt="Screenshot 2024-03-13 at 16 52 20" src="https://github.com/woocommerce/woocommerce/assets/186112/56182d87-7e85-4606-bf24-6db6ffc05e9b">

10. Click on `Design a new theme`.
11. Check there's no 404 error in the inspector and the flow finishes and it loads the assembler (this is the [error](https://github.com/woocommerce/woocommerce/issues/45479) this PR is trying to fix, it should not happen anymore).
13. Go to `wp-admin/themes.php` and make sure the active theme is TT4.


### Changelog entry




<details>

#### Significance




#### Type




#### Message 
CYS - Fix the failed to load resource error in the CYS whenever the current active theme is not TT4
#### Comment 

</details>
